CMakeLists.txt

```
@@ -124,9 +124,13 @@ ELSE()
   # Regular build, not install testing
   # Do all the regular option processing
   IF (NOT KOKKOSKERNELS_HAS_TRILINOS AND NOT KOKKOSKERNELS_HAS_PARENT)
-   # This is a standalone build
-   FIND_PACKAGE(Kokkos REQUIRED)
-   MESSAGE(STATUS "Found Kokkos version ${Kokkos_VERSION} at ${Kokkos_DIR}")
+    # This is a standalone build
+    FIND_PACKAGE(Kokkos REQUIRED)
+    IF((${Kokkos_VERSION} VERSION_EQUAL "4.0.1") OR (${Kokkos_VERSION} VERSION_EQUAL "4.1.00") OR (${Kokkos_VERSION} VERSION_EQUAL "4.1.99"))
```

```suggestion
    IF(${Kokkos_VERSION} VERSION_GREATER "4.0.0")
```

This should be equivalent right? And won't break when Kokkos releases 4.2. Unless we actually want to catch that as an error and update this?

---

In general I think we should only care about the latest patch of a given minor release series since the patch fix bugs?
With that in mind we would have only two version really supported and potentially more versions unsupported in between... So I explicitly pick the two patch versions + develop

---

@lucbv OK, I'm fine with that!